server/src/main/java/org/opensearch/index/IndexSettings.java

```diff
@@ -571,6 +574,18 @@ public final class IndexSettings {
         Property.Dynamic
     );
 
+    public static final Setting<String> INDEX_MERGE_POLICY = Setting.simpleString(
+        "index.merge.policy",
+        DEFAULT_MERGE_POLICY,
+        Property.IndexScope
+    );
+
+    public static final Setting<String> TIME_INDEX_MERGE_POLICY = Setting.simpleString(
+        "indices.time_index.merge.policy",
```

Comment:
```suggestion
        "indices.time_index.default_index_merge_policy"
```

Please also add validation as for `index.merge.policy`, the default value should be `log_byte_size`, right?

Reply:
I'm keeping tiered as default for time_index till we have a benchmark number to support it. I will open another PR to change the default to log_byte_size later.